homeassistant/components/luftdaten/const.py

```diff
@@ -1,10 +1,74 @@
 """Define constants for the Luftdaten component."""
+from __future__ import annotations
+
 from datetime import timedelta
 
+from homeassistant.components.sensor import SensorEntityDescription
+from homeassistant.const import (
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    PERCENTAGE,
+    PRESSURE_HPA,
+    TEMP_CELSIUS,
+)
+
 ATTR_SENSOR_ID = "sensor_id"
 
 CONF_SENSOR_ID = "sensor_id"
 
 DEFAULT_SCAN_INTERVAL = timedelta(minutes=10)
 
 DOMAIN = "luftdaten"
+
+
+SENSOR_HUMIDITY = "humidity"
+SENSOR_PM10 = "P1"
+SENSOR_PM2_5 = "P2"
+SENSOR_PRESSURE = "pressure"
+SENSOR_PRESSURE_AT_SEALEVEL = "pressure_at_sealevel"
+SENSOR_TEMPERATURE = "temperature"
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
```

Comment:
Since you are moving it, move it to sensor.py where it's used.

Reply:
It's used in `__init__` as well. Although not tested, but I believe moving it to `sensors.py` would create a circular dependency.

Reply:
Then I suggest not moving it at all, to keep changes to a minimum

Reply:
If it's not in `sensors.py`, I believe `const.py` is the next best option. However, if you still like me to revert it, I can do that too.

Reply:
please do :+1:

Reply:
👌🏻 Done